packages/rulesets/src/spectral/functions/response-schema-specified-for-success-status-code.ts

```diff
@@ -0,0 +1,19 @@
+export const responseSchemaSpecifiedForSuccessStatusCode = (putOperation: any, _opts: any, ctx: any) => {
+  const errors = []
+  const path = ctx.path
+  const successCodes = ["200", "201"]
+
+  for (const code of successCodes) {
+    const response = putOperation.responses[code]
+    if (!response?.schema) {
```

Comment:
201 is only required in long running put, so need to check if the 'response' is undefined?

Reply:
201 is a valid status code for synchronous PUTs as well.

Reply:
But we should not report error is it's undefined.

Reply:
Thats true. This check should only throw if the schema is undefined "when 201 is present"

Reply:
@rkmanda @jianyexi 
So, we dont have to check for 200 status code having a response schema specified, is that right?

Reply:
You need check the response schema is specified only when the status code is present